lib/postgrex/protocol.ex

```diff
@@ -567,6 +567,7 @@ defmodule Postgrex.Protocol do
 
   defp startup(s, %{opts: opts} = status) do
     params = opts[:parameters] || []
+    params = Keyword.put_new(params, :fallback_application_name, "postgrex")
```

Comment:
Where is `:fallback_application_name` used?

Reply:
I think instead we can always set the application_name name to postgrex unless the user overrides it. No need for the fallback and the application name.

Reply:
well, `:fallback_application_name` is actually a connection parameter recognized by PG--certainly you could structure postgrex to not need it, just using `:application_name` and defaulting as needed--it really comes down to how deeply you want that default embedded, the fallback is more useful if a very low-level lib is likely to be pulled out & re-used elsewhere

Reply:
I see, thanks! It still looks kind of unnecessary given that we can handle this logic ourselves without sending it to the DB, no?

Reply:
yes, I'd be perfectly happy to not use it--setting the default in utils.ex has the added advantage of not spreading the name-assignment logic across multiple locations

Reply:
so do you think it's good for the default to simply be postgrex, or should it pull from application settings somewhere, attempting to be like "postgrex:myapp"???

Reply:
Ok